easybuild/tools/run.py

```diff
@@ -650,14 +650,16 @@ def parse_cmd_output(cmd, stdouterr, ec, simple, log_all, log_ok, regexp):
             _log.debug('cmd "%s" exited with exit code %s and output:\n%s' % (cmd, ec, stdouterr))
 
     # parse the stdout/stderr for errors when strictness dictates this or when regexp is passed in
-    if use_regexp or regexp:
-        res = parse_log_for_error(stdouterr, regexp, msg="Command used: %s" % cmd)
-        if len(res) > 0:
-            message = "Found %s errors in command output (output: %s)" % (len(res), "\n\t".join([r[0] for r in res]))
-            if use_regexp:
-                raise EasyBuildError(message)
+    if fail_on_error_match or regexp:
+        res = parse_log_for_error(stdouterr, regexp, stdout=False)
+        if res:
+            errors = "\n\t" + "\n\t".join([r[0] for r in res])
+            error_str = "error" if len(res) == 1 else "errors"
+            if fail_on_error_match:
+                raise EasyBuildError("Found %s %s in output of %s:%s", len(res), error_str, cmd, errors)
```

Comment:
having only a colon between `cmd` and `errors` makes it difficult to visually detect the separation between the 2.
maybe use something like ` :: ` this instead? (a double colon surrounded by spaces does not occur often and is easy to search for)
```suggestion
                raise EasyBuildError("Found %s %s in output of %s :: %s", len(res), error_str, cmd, errors)
```

Reply:
`errors` starts with a newline (see line 656) so sample output would be:
> ```
> Found 1 error in output of bar:
>    bar is not found
> ```

I think that newline is enough to separate, isn't it?

Reply:
ah, yes indeed.